build_tools/azure/test_script.cmd

```diff
@@ -10,7 +10,7 @@ mkdir %TMP_FOLDER%
 cd %TMP_FOLDER%
 
 if "%CHECK_WARNINGS%" == "true" (
-    set PYTEST_ARGS=%PYTEST_ARGS% -Werror::DeprecationWarning -Werror::FutureWarning
+    set PYTEST_ARGS=%PYTEST_ARGS% -Werror::FutureWarning
```

Comment:
I would still error on `DeprecationWarning` here.

Reply:
I know, but we'd need to fail on a `DeprecationWarning`accidentally raised by us, or by our dependencies (not sure if we do that now though)

Reply:
Ok fair, I'll add them back.